store/postwoman.js

```diff
@@ -102,6 +109,63 @@ export const mutations = {
     settings[key] = value;
   },
 
+  removeVariables({ editingEnvironment }, value) {
+    editingEnvironment.variables = value;
+  },
+
+  setEditingEnvironment(state, value) {
+    state.editingEnvironment = { ...value };
+  },
+
+  setVariableKey({ editingEnvironment }, { index, value }) {
+    editingEnvironment.variables[index].key = value;
+  },
+
+  setVariableValue({ editingEnvironment }, { index, value }) {
+    editingEnvironment.variables[index].value = value;
+  },
+
+  removeVariable({ editingEnvironment }, variables) {
+    editingEnvironment.variables = variables;
+  },
+
+  addVariable({ editingEnvironment }, value) {
```

Comment:
See comment for `setVariableValue`

Reply:
I left addVariable alone since it just automatically creates a basic empty string that can be updated to other types through the setVariableValue